README.md

```diff
@@ -17,7 +17,7 @@ Any other time after the first time:
   - type of ticket / short name of ticket
   - Ex: `feature/project-setup`
 - Commit Messages:
-  - Ticket number: Commit message in past tense
+  - Ticket number: Commit message written in past tense
   - Ex: `AI-16: Added project setup`
 - See for ticket numbers: <https://github.com/users/kayla-himmel/projects/1>
 
```

Comment:
Can we add information somewhere in the README that the site can be accessed at 'localhost:8080' after `node` is run?